### ISSUE TITLE
Extended config docs

### ISSUE THREAD

**Issue by Starkrights** [mahkoh/jay#51]
Hello!

I stumbled across this TWM off of your comment on the bottom of MR !143 on the Wayland GitLab. I've been meaning to get into both Rust and TWMs for a while now, and the fact that configuring Jay is literally writing an entire rust library, _and_ it already supports the soon-to-come wayland fractional scaling? I literally couldn't have asked for a better project to find. 

I had a lot of fun finally reading through rust's ownership system and using the default config, jay-config, and main jay package to . I was giddy at the amount of flexibility jay offers, so I set out to write out some docs for it- both for my own understanding, and the possibility that maybe someone else finds some use out of them. 

That said, I certainly went a bit overboard without dropping an issue here first. I meant to create one earlier, but it was a nice timesink & distraction from my week, and I got carried away. C'est la vie ¯\_(ツ)_/¯

#### TLDR
I started a github wiki with a bit of a guide to configuring jay over on [my fork](https://github.com/StarkRights/jay/wiki). I 100% understand if you don't care/aren't interested though, just wanted to mention it here for the heck of it. 

Best,
Stark

**Comment by mahkoh** [comment]
Looks great! I've opened the wiki here for editing.

Regarding jay_config: This documentation should be generated by cargo doc. I've now configured the documentation to be automatically uploaded to https://mahkoh.github.io/jay/jay_config. If you want to make changes to this documentation, please open a PR.

**Comment by StarkRights** [comment]
Lovely, I'm glad you like it! 

I noticed the source code comments in jay_config, but just hadn't come across the cargo-doc piece of the rust pipeline yet. I'll get a PR rolling for any clarifications I've come up with, clean up the redundant bits of the wiki I started on, and get the docs referenced into the wiki as well.
